splitCheck.py

```diff
@@ -10,62 +10,108 @@
 """
 
 
+from multiprocessing.sharedctypes import Value
 from operator import truediv
+# from __future__ import division
+from datetime import datetime
 import sys
 import keyboard
 
-def namesInput():
-    name_input = input("Enter a name: ")
-    # while True:
-        # name_input = input("Enter a name: ")
-    #     if keyboard.is_pressed('q'):  # if key 'q' is pressed 
-    #             print('You Pressed A Key!')
-    #             break  # finishing the loop
+class MinimumValueError(Exception): pass #handles: no inputs
+class UnwantedStringError(Exception): pass #handles: bad strings in int inputs
+class ErrorInput(Exception): pass #handles: bad inputs in strings
 
-    
+def namesInput(user_total):
+    names_list = []
+    while True:
+        try:
+            name_input = input("Enter a payer's name | Type 'DONE' to proceed: ")
+            if name_input == "DONE":
+                if len(names_list) == 0:
+                    raise MinimumValueError
```

Comment:
keyword 'raise' will make you exit the application? If so, maybe better if we just log the error msg, and have them reenter name, etc... Same for the other 'raise'

Reply:
But doesn't matter too much, I like the exception raises 👍

Reply:
I agree with the first point, normally when something fails, it doesn't crash the program -- but rather inform the user and let them re-try. 

I think removing the `break` keyword solves this...?

Reply:
Lol I didn't see this at all. Raise doesn't crash the program. It does what you think should be done it raises the error and then in one of the exceptions I catch it and prompt them to re-enter until a correct input is given.